extensions/usememos/README.md

```diff
@@ -0,0 +1,14 @@
+# Usememos
+
+[Usememos](https://github.com/zrr1999/usememos) is an extension for [usememos/memos](https://github.com/usememos/memos).
```

Comment:
Link is 404

Reply:
I made a mistake here, it should be https://github.com/zrr1999/raycast-memos. I will make the correction.